### ISSUE TITLE
S1313 FN: VB.NET - No issue is raised if the IP address is assembled as an interpolated string

### ISSUE THREAD

**Issue by csaba-sagi-sonarsource** [SonarSource/sonar-dotnet#5996]
### Description

The VB version of the rule does not report if the IP address is assembled as an interpolated string

### Repro steps

```
        Public Sub StringInterpolation(ByVal unknownPart As String, ByVal knownPart As String)
            Dim part1 As String = "192"
            Dim part2 As String = "168"
            Dim part3 As String = "0"
            Dim part4 As String = "1"
            knownPart = "255"
            Dim ip1 As String = $"{part1}.{part2}.{part3}.{part4}"    ' Noncompliant
            Dim nonIp As String = $"{part1}:{part2}"
            Dim ip2 As String = $"{part1}.{part2}.{part3}.{knownPart}"  ' Noncompliant
            Dim ip3 As String = $"{part1}.{part2}.{part3}.{unknownPart}"
            Dim nestedConstInterpolation As String = $"{$"{part1}.{part2}"}.{part3}.{part4}" ' Noncompliant
            Dim nestedInterpolation As String = $"{$"{part1}.{knownPart}"}.{part3}.{part4}"    ' Noncompliant
        End Sub

```### Expected behavior

The rule should raise an issue.

### Actual behavior

No issues are raised

### Known workarounds

Please provide a description of any known workarounds.

### Related information

* C#/VB.NET Plugins version: 8.42.0.51121
